sql/core/src/main/scala/org/apache/spark/sql/UDFRegistration.scala

```diff
@@ -484,6 +484,21 @@ class UDFRegistration private[sql] (functionRegistry: FunctionRegistry) extends
 
   }
 
+  private[sql] def registerJavaUDAF(name: String, className: String): Unit = {
```

Comment:
It would be good to have ScalaDoc/JavaDoc for this method (similar to the current registerJava function) even though its not external facing, documenting it for whoever needs to update this next will make things easier.